File: learn-js/exercise/random-number.js
function randomNumberGeneratorInRange(rangeStart, rangeEnd) {
  return Math.floor(Math.random() * (rangeEnd - rangeStart + 1)) + rangeStart;
}

console.log(`My random number: ${randomNumberGeneratorInRange(5, 100)}`);
console.log(`My random number: ${randomNumberGeneratorInRange(100, 200)}`);

File: learn-go/challenges/random_number.go
package main

import (
	"fmt"
	"math/rand"
	"time"
)

func randomNumberGeneratorInRange(rangeStart, rangeEnd int) int {
	rand.Seed(time.Now().UnixNano())
	if rangeStart < rangeEnd {
		return rand.Intn(rangeEnd-rangeStart+1) + rangeStart
	}
	return -1
}

func main() {
	fmt.Printf("My random number: %v\n", randomNumberGeneratorInRange(10, 50))
	fmt.Printf("My random number: %v\n", randomNumberGeneratorInRange(100, 200))
	fmt.Printf("My random number: %v\n", randomNumberGeneratorInRange(40, 20))
}
